CONTRIBUTING.md

```diff
@@ -55,6 +55,32 @@ The file should be in 16:9 format, at least 1280 × 720 pixels. To ensure fa
 
 It is strongly recommended to use an image compression tool such as [ImageOptim](https://imageoptim.com) to reduce the image weight. Choosing “lossy” settings at 90% quality will usually yield over 50% compression gains.
 
+## Referencing your package
+
+By referencing your package on the [dedicated page](https://openfisca.org/en/packages/) of the OpenFisca website you give it visibility so that others can use it. We do our best to make this process as easy as possible, [contact us]((mailto:contact@openfisca.org?subject=Referencing%20package)) if you need.
```

Comment:
```suggestion
By referencing your package on the [dedicated page](https://openfisca.org/en/packages/) of the OpenFisca website you give it visibility so that others can use it. We do our best to make this process as easy as possible, [contact us](mailto:contact@openfisca.org?subject=Referencing%20package) if you need.
```